File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int> nums3;
        for (int i = 0; i < nums1.size(); i++) nums3.push_back(nums1[i]);
        for (int i = 0; i < nums2.size(); i++) nums3.push_back(nums2[i]);
        sort(nums3.begin(), nums3.end());
        int x = nums3.size();
        if (x % 2 == 1) return nums3[x / 2];
        else return (nums3[x / 2] + nums3[x / 2 - 1]) / 2.0;
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: list[int], nums2: list[int]) -> float:
        merged_num = nums1 + nums2
        merged_num.sort()
        if len(merged_num)%2 != 0:
            return  merged_num[len(merged_num)//2]
        else:
            return (merged_num[len(merged_num)//2]+merged_num[(len(merged_num)//2)-1])/2    